### DISCUSSION TITLE
how do i move the windows positions using mod chart like the bob onslaught mod

### DISCUSSION THREAD

**Discussion by perngjiafu** [KadeArchive/Kade-Engine#2091] [Q&A] [answered]
i dont understand ._.

**Comment by isakube** [comment, marked as answer]
please dont use a modchart macos users will hate you

`lime.app.Application.current.window.` and use VSCode auto-complete to find `move()` or 'x' and 'y'. this lets you move the window or something
^^^^^^
DONT DO THIS IN LUA, DO IT IN PLAYSTATE IN THE BEATHIT FUNCTION

```haxe
override function beatHit()
{
   // all the other shit
   // just put this at the bottom

   if (curSong.toLowercase() == 'SONG NAME') {
   // stuff
   }
}
```

**Reply by perngjiafu** [reply]
how do i make the window move instead of instantly teleport to the area

**Reply by Riskerman21** [reply]
teleport it 10 pixel at a time really quickly

**Reply by perngjiafu** [reply]
but its every step and i wanna make the window move fast

**Reply by Riskerman21** [reply]
the math just look up a tutorial on how to use haxe math.
thats what the bob onslaught did anyways

**Reply by perngjiafu** [reply]
send me the link

**Reply by Riskerman21** [reply]
https://github.com/wildythomas/mmmbob/blob/main/sourceJon/PlayState.hx
i am not sure what line tho...

**Reply by perngjiafu** [reply]
that doesnt not help :/
im stupid

**Reply by perngjiafu** [reply]
nvm i found it

**Reply by meme286565** [reply]
how do i make it move in a crazy 8 formation like bobs onslaught
